Sources/Sentry/SentryCoreDataTracker.m

```diff
@@ -35,9 +35,18 @@ - (BOOL)managedObjectContext:(NSManagedObjectContext *)context
 
     __block id<SentrySpan> fetchSpan = nil;
     if (context.hasChanges) {
+        __block NSDictionary<NSString *, NSDictionary *> *operations = @{
+            @"INSERTED" : [self countEntities:context.insertedObjects],
+            @"UPDATED" : [self countEntities:context.updatedObjects],
+            @"DELETED" : [self countEntities:context.deletedObjects]
+        };
```

Comment:
`m`: We will send each even if they are empty. Can we only add the ones for which we have entities?